### DISCUSSION TITLE
Missing img shortcode template

### DISCUSSION THREAD

**Discussion by zcliu** [thuliteio/doks#877] [Q&A]
I'm trying to add an image to a page, and got the following error:

Error: Error building site: ".../content/en/tutorials/data/index.md:18:1": failed to extract shortcode: template for shortcode "img-simple" not found

I looked into the /layouts/shortcodes/ folder of the repository, and I don't see img.html or img-simple.html, are these files supposed to be there?

**Comment by h-enk** [comment]
Not anymore, but you could just re-add them from https://github.com/h-enk/doks/tree/v0.3.5/layouts/shortcodes
